resources/dev-tools/faucets.json

```
@@ -23,6 +23,14 @@
             "faucetUrl": "xahau-test.net",
             "shortName": "Xahau-Testnet",
             "desc": "Hooks (L1 smart contracts) enabled Xahau testnet."
+        },
+        {
+            "id": "faucet-select-mpt-devnet",
+            "wsUrl": "wss://mpt.devnet.rippletest.net:51233/",
+            "jsonRpcUrl": "https://wss://mpt.devnet.rippletest.net:51234/",
```

Looks like a typo here.

```suggestion
            "jsonRpcUrl": "https://mpt.devnet.rippletest.net:51234/",
```

---

Yep, caught that one while you were reviewing, I reckon.